resizer/manifest.json

```diff
@@ -11,11 +11,13 @@
       "parameters": [
         {
           "name": "Width",
-          "key": "width"
+          "key": "width",
+          "description": "enter a width in pixels"
```

Comment:
Perhaps we could capitalize the start of the sentence and add a period.

<img width="464" alt="Screen Shot 2021-08-13 at 11 43 35 AM" src="https://user-images.githubusercontent.com/7302036/129384631-e74c64f8-4ede-4803-8116-2a0092065223.png">